custom/ewsghana/reports/specific_reports/reporting_rates.py

```diff
@@ -165,7 +165,7 @@ def rows(self):
                     ReportingRatesReport,
                     self.config['domain'],
                     '?location_id=%s&startdate=%s&enddate=%s',
-                    (values['location_id'], self.config['startdate'], self.config['enddate'])
+                    (values['location_id'], self.config['startdate'].date(), self.config['enddate'].date())
```

Comment:
to make this robust we should call `.strftime('%Y-%m-%d')` on the dates so that we don't rely on the default to string behavior of date()